app/src/main/java/com/quebec/EventDetailsAdding.java

```diff
@@ -0,0 +1,15 @@
+package com.quebec;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class EventDetailsAdding extends AppCompatActivity {
```

Comment:
Change the name of EventDetailsAdding to EventAddDetailsActivity or similar so it is consistent with other activities